pkg/virt-config/configuration_test.go

```diff
@@ -0,0 +1,630 @@
+package virtconfig_test
+
+import (
+	//"encoding/json"
+	//"strings"
+	//"sync"
+
+	"encoding/json"
+	"strings"
+	"sync"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	kubev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/rand"
+	"k8s.io/utils/pointer"
+
+	cmdv1 "kubevirt.io/kubevirt/pkg/handler-launcher-com/cmd/v1"
+	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
+
+	//kubev1 "k8s.io/api/core/v1"
+	//"k8s.io/apimachinery/pkg/api/resource"
+	//metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	//"k8s.io/apimachinery/pkg/util/rand"
+	//"k8s.io/utils/pointer"
+
+	v1 "kubevirt.io/api/core/v1"
+	//cmdv1 "kubevirt.io/kubevirt/pkg/handler-launcher-com/cmd/v1"
+	testutils "kubevirt.io/kubevirt/pkg/testutils"
+	//virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
+)
+
+var _ = Describe("ConfigMap", func() {
```

Comment:
`ConfigMap` does not seem to be accurate anymore :)

Reply:
done